### PR TITLE
Feature/#45 implement tutorial component

### LINKED ISSUE #45: Implement Tutorial Component
More about the task: 
https://www.notion.so/factiverse/d925abb496394a028d6543d38703a11f?v=977a3406b6cb41ca9bca5784ee58af2c&p=ffae0c789ea04ba4946e0022c10130d8

### PR DESCRIPTION
**Pull request by DomyS**
## Related issue

Closes #45

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,8 +0,0 @@
-{
-    // enable globally (here: format on save)
-"editor.formatOnSave": true
-// enable per-language (here: Prettier as formatter)
-"[javascript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
-}
-}
```
Comment (tobiastykvart) [inline .vscode/settings.json:8]: Why did you delete this? Some useful VSCode settings can probably be in the repo.

**Review by tobiastykvart — APPROVED** [review]
LGTM
The only thing I added is `layout: centered` in `Tooltip.stories`, then you can try the different positions of the tooltip with the radio buttons (with the default layout in the top left corner the tooltips are always displayed at the boom or right).

File: .vscode/settings.json (hunk 1 above)
Comment (DomyS) [reply .vscode/settings.json:8]: Hm.. I think I thought that it's not the best idea to have some vs code adjustments in project itself. But if you think  it's alright, let's keep it. 

Is there a reason why you changed button color to primary? Because now for example claim, header, search has everywhere white button, where it should be yellow

Comment (tobiastykvart) [reply .vscode/settings.json:8]: I'm thinking we should use the yellow more sparingly, to draw the user's attention. Something like the example claims can have the white button.

Comment (DomyS) [reply .vscode/settings.json]: So decided to leave colors as it is, I mean with your changes. If I understood correctly according to new design all components that I've mentioned should change as well with it. And maybe it would be easier to agree with colors when we will have it in the big picture.

**Review by tobiastykvart — APPROVED** [review]
